src/atlantis/manager/rpc/ldap.go

```diff
@@ -263,6 +264,12 @@ func (e *AddTeamAdminExecutor) Authorize() error {
 	if err != nil || !res.IsAdmin {
 		return errors.New("Permission denied")
 	}
+        reql := ManagerListTeamAdminsArg{e.arg.ManagerAuthArg, e.arg.Team}
+        var resl  ManagerListTeamAdminsReply
+        err = NewTask("AddTeamAdmin-ListTeamAdmins", &ListTeamAdminsExecutor{reql, &resl}).Run()
+        if err != nil || len(resl.TeamAdmins) >= maxTeamAdmins {
```

Comment:
Please don't ignore errors; if, for example, the LDAP server is down, this will give the wrong message.  If there's an error, pass it along, don't create a new one completely unrelated to the actual problem.